src/test/java/tutorspet/logic/parser/DeleteAttendanceCommandParserTest.java

```diff
@@ -27,93 +31,161 @@
 
     @Test
     public void parse_validArgs_returnsDeleteAttendanceCommand() {
-        assertParseSuccess(parser, " c/1 l/1 s/1 w/1",
+        assertParseSuccess(parser, " "
+                + PREFIX_CLASS_INDEX + "1" + " "
+                + PREFIX_LESSON_INDEX + "1" + " "
+                + PREFIX_STUDENT_INDEX + "1" + " "
+                + PREFIX_WEEK + "1",
                 new DeleteAttendanceCommand(INDEX_FIRST_ITEM, INDEX_FIRST_ITEM, INDEX_FIRST_ITEM, VALID_WEEK_1));
 
         // multiple class indexes -> last class index accepted
-        assertParseSuccess(parser, " c/1 l/1 c/2 s/1 w/1",
+        assertParseSuccess(parser, " "
+                + PREFIX_CLASS_INDEX + "1" + " "
+                + PREFIX_LESSON_INDEX + "1" + " "
+                + PREFIX_CLASS_INDEX + "2" + " "
+                + PREFIX_STUDENT_INDEX + "1" + " "
+                + PREFIX_WEEK + "1",
                 new DeleteAttendanceCommand(INDEX_SECOND_ITEM, INDEX_FIRST_ITEM, INDEX_FIRST_ITEM, VALID_WEEK_1));
 
         // multiple lesson indexes -> last lesson index accepted
-        assertParseSuccess(parser, " c/1 l/1 s/1 l/2 w/1",
+        assertParseSuccess(parser, " "
+                + PREFIX_CLASS_INDEX + "1" + " "
+                + PREFIX_LESSON_INDEX + "1" + " "
+                + PREFIX_STUDENT_INDEX + "1" + " "
+                + PREFIX_LESSON_INDEX + "2" + " "
+                + PREFIX_WEEK + "1",
                 new DeleteAttendanceCommand(INDEX_FIRST_ITEM, INDEX_SECOND_ITEM, INDEX_FIRST_ITEM, VALID_WEEK_1));
 
         // multiple student indexes -> last student index accepted
-        assertParseSuccess(parser, " c/1 l/1 s/1 w/1 s/2",
+        assertParseSuccess(parser, " "
+                + PREFIX_CLASS_INDEX + "1" + " "
+                + PREFIX_LESSON_INDEX + "1" + " "
+                + PREFIX_STUDENT_INDEX + "1" + " "
+                + PREFIX_WEEK + "1" + " "
+                + PREFIX_STUDENT_INDEX + "2",
                 new DeleteAttendanceCommand(INDEX_FIRST_ITEM, INDEX_FIRST_ITEM, INDEX_SECOND_ITEM, VALID_WEEK_1));
 
-        assertParseSuccess(parser, " c/1 l/1 s/1 w/1 w/5",
+        assertParseSuccess(parser, " "
+                + PREFIX_CLASS_INDEX + "1" + " "
+                + PREFIX_LESSON_INDEX + "1" + " "
+                + PREFIX_STUDENT_INDEX + "1" + " "
+                + PREFIX_WEEK + "1" + " "
+                + PREFIX_WEEK + "5",
                 new DeleteAttendanceCommand(INDEX_FIRST_ITEM, INDEX_FIRST_ITEM, INDEX_FIRST_ITEM, VALID_WEEK_5));
     }
 
     @Test
     public void parse_missingClassIndex_throwsParseException() {
-        assertParseFailure(parser, " l/1 s/1 w/1", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+        assertParseFailure(parser, " "
+                + PREFIX_LESSON_INDEX + "1" + " "
+                + PREFIX_STUDENT_INDEX + "1" + " "
+                + PREFIX_WEEK + "1", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                 MESSAGE_USAGE));
```

Comment:
For statements such as these, would it be better to put the `String.format` method on the next line?
```suggestion
        assertParseFailure(parser, " "
                + PREFIX_LESSON_INDEX + "1" + " "
                + PREFIX_STUDENT_INDEX + "1" + " "
                + PREFIX_WEEK + "1", 
                String.format(MESSAGE_INVALID_COMMAND_FORMAT, MESSAGE_USAGE));
```